druntime/src/core/sys/posix/fcntl.d

```diff
@@ -414,11 +414,10 @@ version (linux)
         enum O_DIRECTORY    = 0x010000; // octal    200000
         enum O_NOFOLLOW     = 0x020000; // octal    400000
         enum O_DIRECT       = 0x004000; // octal    040000
-        version (D_LP64)
-            enum O_LARGEFILE = 0;
-        else
-            enum O_LARGEFILE = 0x8000;  // octal   0100000
-        enum O_TMPFILE      = 0x404000; // octal 020040000
+
+        enum O_LARGEFILE = 0;
+
+        enum O_TMPFILE      = 0x410000; // octal 020200000
```

Comment:
ditto

Reply:
and

```gcc -E -P -D_GNU_SOURCE -include fcntl.h - <<< "O_TMPFILE"```

Reply:
Preprocessing headers doesn't give an accurate representation of what the definition should be. There are systems which have been compiled with LARGEFILE.

Reply:
I assume `version (LoongArch64)` implies `version (D_LP64)`. So does the other case seem unnecessary?